### ISSUE TITLE
NPM module

### ISSUE THREAD

**Issue by p0o** [mymonero/mymonero-core-js#57]
Is it possible to have an NPM module published for this repository? Preferably with a babel build script to publish the ES5 code so it wouldn't be rejected by React build environments.

**Comment by paulshapiro** [comment]
You can already install it over npm by specifying the git repo as there's a package.json and index. 

I would generally be fine with the npm registry listing this repo but just be aware that you run a security risk by having that layer of automation... something I would tend to try to avoid recommending for such a sensitive library.

**Comment by p0o** [comment]
Yes, you are right about installing git repo using NPM, however, the master branch is not ES5 friendly (needs a bable build with a dist folder) and I'm not sure if cpp branch is production ready?

**Comment by paulshapiro** [comment]
The master branch has been built to es5 successfully by many people…

Cpp is close to production ready. More people should test it.

**Comment by paulshapiro** [comment]
What would be done to the cpp branch which could not be done to the master branch anyway? I would accept a working PR .

**Comment by p0o** [comment]
> The master branch has been built to es5 successfully by many people…

Yes I also put it in my webpack build but it's adding extra time to my environment build time and many standard webpack builds would ignore `node_modules` folder for build. It's common for JS dependencies to include a dist folder for prebuilt files to avoid these kind of issues.

**Comment by paulshapiro** [comment]
Feel free to submit fixes via PR. This is a community project :)

**Comment by p0o** [comment]
Sure, will submit soon
